.gitignore

```diff
@@ -9,3 +9,5 @@ Cargo.lock
 # These are backup files generated by rustfmt
 **/*.rs.bk
 
+# This folder contains "ideas" for the IntelliJ workspace
+.idea/
```

Comment:
Just FYI: Intellij can exclude this for you without including it in gitignore. I don't think we should care about it but some projects don't want IDE specific ignores here.

Reply:
You're right. I deleted it 👍